classes/village/VillageApiPresenter.php

```diff
@@ -161,6 +161,84 @@ function (WarLogDto $war_log) {
             "player_war_log" => $player_war_log
         ];
     }
+    public static function warRecordDataResponse(System $system, User $player, int $page_number = 1): array
+    {
+        $war_records = array_map(
+            function (WarRecordDto $warRecordDto) {
+                return [
```

Comment:
Not a big deal but if we're immediately returning the value of `$war_records`, we could skip a step and return it directly
```suggestion
        return [
            "war_records" => array_map(
                function (WarRecordDto $warRecordDto) {
                    return [
```